translate/test/translate.test.js

```diff
@@ -114,12 +115,43 @@ describe('translate:translate', function () {
     });
   });
 
+  describe('listLanguageNames', function () {
+    it('should list languages', function () {
+      var sample = getSample();
+      var callback = sinon.stub();
+
+      sample.program.listLanguageNames(target, apiKey, callback);
+
+      assert(sample.mocks.translate.getLanguages.calledOnce, 'method called once');
+      assert.equal(sample.mocks.translate.getLanguages.firstCall.args.length, 2, 'method received 2 arguments');
+      assert(callback.calledOnce, 'callback called once');
+      assert.equal(callback.firstCall.args.length, 2, 'callback received 2 arguments');
```

Comment:
This looks like the old `assert` style - mind upgrading it to the newer "gold standard" in-place?